views/components/form/checkbox.erb

```diff
@@ -12,10 +12,11 @@ attributes ||= {} %>
   <fieldset>
     <div class="space-y-5">
       <% options.each do |opt_val, opt_text, opt_classes, opt_attrs| %>
+        <% id_suffix = SecureRandom.alphanumeric(6) %>
```

Comment:
nit: We could use the index of the options instead of a random suffix

Reply:
Makes sense. Updated it.